pkg/controllers/podgroup/pg_controller.go

```
@@ -150,3 +157,32 @@ func contains(slice []string, element string) bool {
 	}
 	return false
 }
+
+func (pg *pgcontroller) deleteWorker() {
+	for pg.processNextWorkItem() {
+	}
+}
+
+func (pg *pgcontroller) processNextWorkItem() bool {
+	obj, shutdown := pg.pgDeletedQueue.Get()
+	if shutdown {
+		klog.Errorf("Fail to pop item from queue")
+		return false
+	}
+
+	req := obj.(pgRequest)
+	defer pg.pgDeletedQueue.Done(req)
+
+	if _, err := pg.pgLister.PodGroups(req.pgNamespace).Get(req.pgName); err != nil {
```

Did we ignore this error?